app/views/vendors/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1> Welcome, <%=@market.name%>! Edit a vendor: </h1>
+
+
+<% if @vendor.errors.any? %>
+  <ul>
+    <% @vendor.errors.each do |column, message| %>
+      <li>
+        <%= column.capitalize %> <%= message %>
+      </li>
+    </ul>
+    <% end %>
+<% end %>
+
+
+<%=render partial: 'form'%>
```

Comment:
Nice use of a partial here for the edit and new forms! You can also use a partial for your errors to use in all of your views.